eng/linkerTests/SupportFiles/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
```

Comment:
nit: the naming is confusing this is not testing linker but libraries "linkability"

Reply:
Your comment makes sense, I decided to use LinkerTests since they are basically running the linker and then checking the output. I can rename them if preferred. Would your suggestion be "LinkabilityTests"?

Reply:
Assuming you are going to test correctness only. I'd call it for example `TrimmingTests`